scripts/get-helm-3

```diff
@@ -94,7 +93,7 @@ checkDesiredVersion() {
 # if it needs to be changed.
 checkHelmInstalledVersion() {
   if [[ -f "${HELM_INSTALL_DIR}/${PROJECT_NAME}" ]]; then
-    local version=$("${HELM_INSTALL_DIR}/${PROJECT_NAME}" version -c | grep '^Client' | cut -d'"' -f2)
+    local version=$("${HELM_INSTALL_DIR}/${PROJECT_NAME}" version --template="{{ .Version }}")
```

Comment:
So now I'm probably pushing it...
The first install of v3 will normally be with a current installation of v2.  Two things:
1. `--template="{{ .Version }}"` returns `<no value>` for v2.  Not a real problem since it will still go ahead with the install of v3 (there will be a printout to the user saying `Helm canary is available. Changing from version <no value>.`)
1. if the cluster is not reachable, `helm2 version` will take a long time.  It would be nice to have the `-c` flag like before.

Reply:
Good call. Whatever fix we apply to `get`'s version selection filter should probably be applied here as well. That way this script cannot be used with Helm 2, and vice versa.

Reply:
As for

> The first install of v3 will normally be with a current installation of v2

This will be fixed once Helm 3 has been released, and given this script is called `get-helm-3`, hopefully it's clear to users that this should only be used for fetching Helm 3 release binaries. :)

Reply:
> This will be fixed once Helm 3 has been released, and given this script is called `get-helm-3`, hopefully it's clear to users that this should only be used for fetching Helm 3 release binaries. :)

Right. So we can assume the user will be downloading v3. But the first time they do so there will probably be a v2 currently installed on their machine (that was my case when I tried it). IIUC `checkHelmInstalledVersion()` checks the *currently* installed version, so this call will run with v2.

Reply:
Ah! I see where you're getting at now. Thank you for clarifying.

Reply:
That being said, things do work as needed as they stand.